src/components/AttachmentPicker/index.native.js

```diff
@@ -250,136 +148,207 @@ class AttachmentPicker extends Component {
 
                 return resolve(response.assets);
             });
-        });
-    }
-
-    /**
-     * A generic handling when we don't know the exact reason for an error
-     *
-     */
-    showGeneralAlert() {
-        Alert.alert(this.props.translate('attachmentPicker.attachmentError'), this.props.translate('attachmentPicker.errorWhileSelectingAttachment'));
-    }
+        })
+
+    const [menuItemData, setMenuItemData] = useState([
```

Comment:
Can this be moved up top so it stays closer to other state statements? I think it's easier to read this way 😄

Reply:
Deprecated as we will convert this into a useMemo